module/cloud-services/files/helper.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * name : files/helper.js
+ * author : prajwal
+ * created-date : 25-Apr-2024
+ * Description : All files related helper functionality.Including uploading file
+ * to cloud service.
+ */
+
+// Dependencies
+
+let filesHelpers = require(MODULES_BASE_PATH + "/files/helper");
+const cloudStorage = process.env.CLOUD_STORAGE_PROVIDER;
+const bucketName = process.env.CLOUD_STORAGE_BUCKETNAME;
+const bucktType = process.env.CLOUD_STORAGE_BUCKET_TYPE
+const fs = require("fs");
+
+
+/**
+ * FilesHelper
+ * @class
+ */
+
+module.exports = class FilesHelper {
+  /**
+   * Get all signed urls.
+   * @method
+   * @name preSignedUrls
+   * @param {Array} payloadData       - payload for files data.
+   * @param {String} referenceType    - reference type
+   * @param {String} userId           - Logged in user id.
+   * @param {String} templateId       - certificateTemplateId.
+   * @param {Boolean} serviceUpload     - serive Upload  {true/false}
+   * @returns {Array}                 - consists of all signed urls files.
+   */
+
+  static preSignedUrls(payloadData, userId = "") {
```

Comment:
@Prajwal17Tunerlabs Add proper comments to increase readability of the code

Reply:
added comments to increase readability of the code